Support coe font studio in20241215

Rintaro1234 opened the pull request:
お世話になっております．
自分の環境のみかもしれませんが，CoeFontStudioのクラス名が変わったようで動かなくなっていたのを直しました．

※二回目のプルリクとなります．一回目は.vscodeファイルが混ざっていたので修正しました．

確認していただければ幸いです．

Rintaro1234 committed:
[fix]2024年12月15日のCoeFontStudioに対応させた

Rintaro1234 committed:
[fix].vscodeファイルが残っていたので消した．

oov commented on content-script.js:
```diff
@@ -20,7 +20,7 @@ function getName(blockRoot) {
 
 document.body.addEventListener('click', e => {
-  const button = e.target.closest('[class*=IconButton_button__]:nth-of-type(2)');
+  const button = e.target.closest('[class*=IconButton_button__]:nth-of-type(1)');
```
この変更で上手く動作するのを確認しました、ありがとうございます。

ただ、以前このボタンには固有と思われる ID や識別子などが特になかったため苦肉の策として `nth-of-type` を使用していたのですが、現在では PC とモバイルの両方で `editor_sentence_download_btn` という ID がエレメントに割り当てられているようです。
```js
const button = e.target.closest('#editor_sentence_download_btn');
```
こちらのほうが、構造が多少変更された場合や他の拡張機能でエレメントが追加されていた場合などにも上手く動く可能性が少し高くなると思いますので、ID での指定に変更していただけますか？

oov reviewed (commented):
PR ありがとうございます、大変助かりました。
レビューさせていただきましたのでご確認よろしくお願いいたします。

なお今回の PR では変更されていない場所なのですが、現在のサイトにおいては不要になった処理がありました。

CoeFont の編集画面には画面幅によって PC 向けとモバイル向けでレイアウトが変更される仕組みがあり、以前はこれが内部的には別々に実装されていたのですが、現在ではおそらく共通化されたようです。
https://github.com/oov/coefont_autotxtdl/blob/05275cf8934e428d7bfee5834c347315f390c3d7/content-script.js#L14-L18
このチェックは現在は不要になっているようなので、合わせて削除していただけますと嬉しいです。
お手数をおかけしますがよろしくお願いいたします。

Rintaro1234 committed:
[fix]不要になったgetName関数内の画面幅がモバイルのときの特別な処理を削除

[fix]ボタンが押された親ウィジェットの取得方法を，クラス名の検索からIDの検索に変更
[fix]テキストボックスに何も入っていない状態でダウンロードボタンを押したあと，今度は入力されているダウンロードボタンを押すとスクリプトが動かなくなるバグを修正，方法は何も入っていない場合（='\n'）は処理を抜けるように変更

oov commented:
PRありがとうございました。
大変助かりました。
